sdk/documentintelligence/azure-ai-documentintelligence/azure/ai/documentintelligence/models/_models.py

```diff
@@ -1726,21 +1727,21 @@ class DocumentList(_model_base.Model):
 
     :ivar spans: Location of the list in the reading order concatenated content. Required.
     :vartype spans: list[~azure.ai.documentintelligence.models.DocumentSpan]
-    :ivar items: Items in the list. Required.
-    :vartype items: list[~azure.ai.documentintelligence.models.DocumentListItem]
+    :ivar items_property: Items in the list. Required.
+    :vartype items_property: list[~azure.ai.documentintelligence.models.DocumentListItem]
     """
 
     spans: List["_models.DocumentSpan"] = rest_field()
     """Location of the list in the reading order concatenated content. Required."""
-    items: List["_models.DocumentListItem"] = rest_field()
+    items_property: List["_models.DocumentListItem"] = rest_field(name="items")
```

Comment:
The generator had to rename this because `items` conflicts with the base model MutableMapping `items`. Are you okay with this name or do you want to rename it in the TypeSpec?

Reply:
Let's keep name for now, I'll ask the rename in v-team.

Reply:
You might want to add this rename to the changelog in the meantime, so we don't forget

Reply:
Yup, have mentioned it in changelog.